tools/migrate_gid_to_uid/migrate_gid_to_uid.go

```diff
@@ -272,28 +296,60 @@ func buildGroupIDToUserIDMap(logger log.Logger, config *Config, metaIDToUserIDMa
 		Value  string `bson:"value"`
 	}
 	for iter.Next(&result) {
-		if metaID := result.MetaID; metaID == "" {
-			logger.Warn("Missing meta id in result from meta query")
-		} else if result.Value == "" {
-			logger.WithField("metaId", metaID).Warn("Missing value in result from meta query for meta id")
-		} else {
-			var value struct {
-				Private struct {
-					Uploads struct {
-						ID string `json:"id"`
-					} `json:"uploads"`
-				} `json:"private"`
-			}
-			if err = json.Unmarshal([]byte(result.Value), &value); err != nil {
-				logger.WithField("metaId", metaID).WithError(err).Warn("Unable to unmarshal value from meta query for meta id")
-			} else if groupID := value.Private.Uploads.ID; groupID == "" {
-				logger.WithField("metaId", metaID).Debug("Missing private uploads id in value in result from meta query for meta id")
-			} else if userID, ok := metaIDToUserIDMap[metaID]; !ok {
-				logger.WithField("metaId", metaID).Warn("Missing user id for meta id")
-			} else {
-				groupIDToUserIDMap[groupID] = userID
-			}
+		metaLogger := logger
+
+		metaID := result.MetaID
+		if metaID == "" {
+			metaLogger.Warn("Missing meta id in result from meta query")
+			continue
+		}
+
+		metaLogger = metaLogger.WithField("metaId", metaID)
```

Comment:
same 'issue' as userID - if the metaID is missing and we then setup the logger with an empty metaID will that be confusing?